functions/rust-mnist/www/.gitignore

```diff
@@ -0,0 +1,24 @@
+# Logs
+logs
+*.log
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
+pnpm-debug.log*
+lerna-debug.log*
```

Comment:
Do we need to list all the `xyz-debug.log` files? Will they be caught by the `*.log` entry, or do they have a suffix?

Reply:
Thank you, I didn't notice. This was generated by the vite CLI (https://vitejs.dev/) which is a development server for front-end applications.